src/common/CollabsibleCard.tsx

```diff
@@ -15,14 +15,15 @@ function CollapseIndicator({ open }: { open: boolean }): JSX.Element {
     />
   )
 }
-
-export class CollapsibleCard extends Component<
-  { label; children; collapsed: boolean },
-  { open: boolean }
-> {
+interface Props {
+  label: JSX.Element | null
+  children: ReactNode
```

Comment:
Why label is `JSX.Element | null` but children is `ReactNode`?
See: https://github.com/typescript-cheatsheets/react#useful-react-prop-type-examples